libs/designsystem/src/lib/components/flag/flag.component.ts

```diff
@@ -1,10 +1,8 @@
-import { Component, Input, ChangeDetectionStrategy, HostBinding } from '@angular/core';
+import { ChangeDetectionStrategy, Component, HostBinding, Input } from '@angular/core';
 
 @Component({
   selector: 'kirby-flag',
-  template: `
-    <ng-content></ng-content>
-  `,
+  template: ` <ng-content></ng-content> `,
```

Comment:
I suppose the spaces don't make any difference. Can you confirm?

Reply:
Confirmed!